torch/nn/modules/activation.py

```diff
@@ -726,6 +726,12 @@ def _reset_parameters(self):
         if self.bias_v is not None:
             xavier_normal_(self.bias_v)
 
+    def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
+                              missing_keys, unexpected_keys, error_msgs):
+        super(MultiheadAttention, self)._load_from_state_dict(
```

Comment:
Just forwarding a call to the `super` shouldn't do anything extra, shouldn't this be adding `_qkv_same_embed_dim` to the `state_dict` with a default value if it's not present in `state_dict` already (meaning the state dict is from an older model) so the user doesn't have to do anything?

Reply:
No. _qkv_same_embed_dim should not be in state_dict. state_dict should only have learnable parameter.
In this case, state_dict tracks in_proj_weight/in_proj_bias/out_proj.weight/out_proj.bias.